src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -687,7 +674,7 @@ public String validateGiverRecipientVisibility(FeedbackQuestionAttributes feedba
     }
 
     public int getNumOfMsqChoices() {
```

Comment:
Can we remove this method? The place that calls it can just call `getMsqChoices().size()` instead.